CMakeLists.txt

```diff
@@ -133,11 +133,18 @@ if(WARNING_AS_ERROR)
     endif()
 endif()
 
+if(BUILD_TESTING)
+    enable_testing()
+endif()
```

Comment:
Nice. We used `BUILD_TEST` in iceoryx classic and I changed the flag for iceoryx2 to be more in line with the cmake ecosystem. It seems I forgot about this. Btw, we are in the process to make the C and C++ bindings build standalone for packagers. Do you know if it is a problem to cal `enable_testing()` multple times?

Reply:
It should be fine to invoke it multiple times. Basically, you will want one invocation for every directory level where you want `ctest` to work, so usually the `CMakeLists.txt` with a `project` command will also end up with an `enable_testing`.